app/views/result_filters/subject/_form.html.erb

```diff
@@ -57,19 +57,19 @@
                             <% end %>
                           </span>
 
-                          <% if subject.decorate.has_scholarship_and_bursary? %>
-                            <% financial_info = "Scholarships of £#{number_with_delimiter(subject.scholarship, :delimiter => ',')} and bursaries of £#{number_with_delimiter(subject.bursary_amount, :delimiter => ',')} are available" %>
-                          <% elsif subject.decorate.has_scholarship? %>
-                            <% financial_info = "Scholarships of £#{number_with_delimiter(subject.scholarship, :delimiter => ',')} are available" %>
-                          <% elsif subject.decorate.has_bursary? %>
-                            <% financial_info = "Bursaries of £#{number_with_delimiter(subject.bursary_amount, :delimiter => ',')} available" %>
-                          <% end %>
-                          <% if subject.decorate.early_career_payments? %>
-                            <% financial_info.concat(", \n with early career payments of £2,000 each in your second, third and fourth year of teaching (£3,000 in some areas of England).") %>
-                          <% elsif subject.decorate.has_scholarship? || subject.decorate.has_bursary? %>
-                            <% financial_info.concat(".") %>
-                          <% end %>
-                          <span class="govuk-body" data-qa="subject__info"><%= financial_info %></span>
+                          <%# if subject.decorate.has_scholarship_and_bursary? %>
+                            <%# financial_info = "Scholarships of £#{number_with_delimiter(subject.scholarship, :delimiter => ',')} and bursaries of £#{number_with_delimiter(subject.bursary_amount, :delimiter => ',')} are available" %>
+                          <%# elsif subject.decorate.has_scholarship? %>
+                            <%# financial_info = "Scholarships of £#{number_with_delimiter(subject.scholarship, :delimiter => ',')} are available" %>
+                          <%# elsif subject.decorate.has_bursary? %>
+                            <%# financial_info = "Bursaries of £#{number_with_delimiter(subject.bursary_amount, :delimiter => ',')} available" %>
+                          <%# end %>
+                          <%# if subject.decorate.early_career_payments? %>
+                            <%# financial_info.concat(", \n with early career payments of £2,000 each in your second, third and fourth year of teaching (£3,000 in some areas of England).") %>
+                          <%# elsif subject.decorate.has_scholarship? || subject.decorate.has_bursary? %>
+                            <%# financial_info.concat(".") %>
+                          <%# end %>
+<!--                          <span class="govuk-body" data-qa="subject__info"><%#= financial_info %></span>-->
 
                           <% if subject.subject_knowledge_enhancement_course_available %>
```

Comment:
and this bit

![image](https://user-images.githubusercontent.com/470137/95177721-70a16600-07b6-11eb-9d09-d0af1e279dcb.png)

Reply:
Cheers @defong - making the change